crates/gpui/src/platform/windows/text_system.rs

```diff
@@ -0,0 +1,440 @@
+use crate::{
```

Comment:
haven't reviewed this, assuming it's just a fork of LinuxTextSystem.

Reply:
actually, I take it back. Maybe it's totally fine to provide the text as a copy. We may still want to consider other approaches here, and moving making cosmic text available everywhere would be premature.